### PR TITLE
[CS-4443] Adding salesforce as one of the providers

### PR DESCRIPTION
**Pull request by shwetai**

### CODE REVIEW

**Comment by sleslie23** [comment]
Thanks @shwetai !!!

File: access-management/v/latest/external-identity.adoc [hunk 1]
```diff
@@ -17,18 +17,28 @@ The following diagram shows the SAML identity management process:
 
 image:external-identity-decbd.png[external-identity-decbd]
 
-Before configuring OpenID Connect or SAML 2.0, select an identity management provider (IdP). If you intend to use OpenID Connect, choose an OpenID Connect-compliant provider, such as PingFederate, OpenAM, or Okta; otherwise, choose any SAML 2.0-compliant provider. The following SAML providers have been successfully tested with Anypoint Platform:
+Before configuring OpenID Connect or SAML 2.0, select an identity management provider (IdP). To use OpenID Connect, select an OpenID Connect-compliant provider, such as PingFederate, OpenAM, or Okta. If you are not using OpenID Connect, select any SAML 2.0-compliant provider.
 
+The following SAML providers have been tested on Anypoint Platform and are fully supported:
```
Comment (sebastiankorol) [inline access-management/v/latest/external-identity.adoc]: Remove "have been tested on Anypoint Platform"

File: access-management/v/latest/external-identity.adoc [hunk 2]
```diff
@@ -17,18 +17,28 @@ The following diagram shows the SAML identity management process:
...
 * PingFederate
 * OpenAM
 * Okta
-* Shibboleth
+* Salesforce
```
Comment (sebastiankorol) [inline access-management/v/latest/external-identity.adoc]: List Salesforce first

File: access-management/v/latest/external-identity.adoc [hunk 3]
```diff
@@ -17,18 +17,28 @@ The following diagram shows the SAML identity management process:
...
 * Active Directory Federation Services (AD FS)
+
+These providers work with Anypoint Platform, but have not been fully tested :
```
Comment (sebastiankorol) [inline access-management/v/latest/external-identity.adoc]: The following SAML providers are known to work, but aren't actively tested

File: access-management/v/latest/external-identity.adoc [hunk 4]
```diff
@@ -17,18 +17,28 @@ The following diagram shows the SAML identity management process:
 
 image:external-identity-decbd.png[external-identity-decbd]
 
-Before configuring OpenID Connect or SAML 2.0, select an identity management provider (IdP). If you intend to use OpenID Connect, choose an OpenID Connect-compliant provider, such as PingFederate, OpenAM, or Okta; otherwise, choose any SAML 2.0-compliant provider. The following SAML providers have been successfully tested with Anypoint Platform:
+Before configuring OpenID Connect or SAML 2.0, select an identity management provider (IdP). To use OpenID Connect, select an OpenID Connect-compliant provider, such as PingFederate, OpenAM, or Okta. If you are not using OpenID Connect, select any SAML 2.0-compliant provider.
 
+The following SAML providers have been tested on Anypoint Platform and are fully supported:
 * PingFederate
 * OpenAM
```
Comment (sebastiankorol) [inline access-management/v/latest/external-identity.adoc]: Add versions

File: access-management/v/latest/external-identity.adoc [hunk 5]
```diff
@@ -17,18 +17,28 @@ The following diagram shows the SAML identity management process:
...
+
+These providers work with Anypoint Platform, but have not been fully tested :
+* Shibboleth
 * onelogin
 * CA Single Sign-On
 * SecureAuth
 
-Familiarize yourself with the documentation of your Idp. After you select an IdP, set up your Anypoint Platform organization as your audience in your IdP configuration. Configure identity management in the Anypoint Platform master organization. The IdP you select is effective for the entire organization and all business groups. Configure attribute names on the IdP and Anypoint Platform to match.
+
+The following providers have been tested on Anypoint Platform and are fully supported:
```
Comment (sebastiankorol) [inline access-management/v/latest/external-identity.adoc]: Same comments as above

**Pull request merged**